recipes-graphics/xorg-xserver/xserver-xf86-config_%.bbappend

```diff
@@ -4,6 +4,11 @@ SRC_URI:append:rpi = " \
     file://xorg.conf.d/98-pitft.conf \
     file://xorg.conf.d/99-calibration.conf \
 "
+
+SRC_URI:append:raspberrypi5 = " \
+    file://99-vc4.conf \
```

Comment:
does it need to be pi5 specific ?

Reply:
This is needed for (at least) both pi5 and cm5 with I/O board. Using this in something like raspberrypi5.conf allows to do `SRC_URI:append:pi5` at least.

```bitbake
OVERRIDES:append = ":hwpi5:pi5"
```

However, from what I have read vc4 may apply to pi4/cm4-io (at least) as well. One could do something like

```bitbake
OVERRIDES:append = ":pi-vc4"
```

in an include file for any Pi board to which this applies (e.g. probably not the Zero/Zero W)
and `SRC_URI:append:pi-vc4` here.

The official Pi hardware documentation might have the info needed to determine which models this is needed for.